src/main/g8/README.md

```diff
@@ -196,18 +207,19 @@ object Main {
 
 ## [scala-logging](https://github.com/lightbend/scala-logging) - universal logging ##
 
-`scala-logging` is a logging adapter specific to Scala that uses the [slf4j logging interface](https://www.slf4j.org/manual.html).  In turn, that interface is hooked up to [Log4J as the logging mechanism](http://logging.apache.org/log4j/2.x/manual/layouts.html) (the thing that actually logs), which has been sensibly preconfigured.
+`scala-logging` is a logging adapter specific to Scala that uses the [slf4j logging interface](https://www.slf4j.org/manual.html).  In turn, that interface is hooked up to [logback as the logging mechanism](http://logback.qos.ch/) (the thing that actually logs), which has been sensibly preconfigured.
 
 ###### Why do we use it? ######
 
 - `scala-logging` gives you a limited subset of the functionality (only the stuff you need).
 - `scala-logging` is coupled tightly to scala, and thus is very performant, making use of scala macros to intelligently not log when the logging level is not requested.  This allows us to leave trace & debug logs everywhere we need them without worrying about performance.
 - Abstraction over `slf4j` allows the end-user to decide how they want to handle logs.
-- `Log4J2` can be drop-in-replaced - all of the dependencies are run-time.  If you want to use logback (or whatever logging mechanism), simply bring along the dependency in the environment you wish to run in, and configure it.
+- `logback-classic` can be drop-in-replaced - all of the dependencies are run-time.
+- If you want to use `Log4J2` (or another logging backend that relies heavily upon reflection), be warned that the SubstrateVM's "out of the box" configuration won't be enough and [you'll need to do some configuration first](https://github.com/oracle/graal/issues/808#issuecomment-470769285). If you don't absolutely need log4j2, you could simply wait for SubstrateVM's [automatic detection of reflection](https://github.com/oracle/graal/blob/master/substratevm/REFLECTION.md#automatic-detection) to be expanded upon so you can keep your project a little more lean in the meantime.     
```

Comment:
Now **that's** a spicy meatball.  

In the spirit of [your comment](https://github.com/sbchapin/quickstart-scala-sbt-native-image.g8/pull/1#discussion_r348871033), might I suggest adding a quick callout to [this area of `build.sbt`](https://github.com/sbchapin/quickstart-scala-sbt-native-image.g8/blob/master/src/main/g8/build.sbt#L29-L31) in the form of a small comment that says `// See README.md why we don't log4j2`

Reply:
![tenor](https://user-images.githubusercontent.com/20078622/69367137-8b292a00-0c4c-11ea-8729-1839c6b232b6.gif)